.travis.yml

```diff
@@ -1,5 +1,12 @@
-sudo: false
+sudo: true
```

Comment:
this could be improved by submitting @raphink's ppa to https://github.com/travis-ci/apt-source-whitelist